cannon/mipsevm/tests/evm_common_test.go

```diff
@@ -499,10 +499,11 @@ func TestEVMFault(t *testing.T) {
 		name   string
 		nextPC uint32
 		insn   uint32
+		errMsg string
```

Comment:
I would configure the memory proof addresses in the test case instead of hardcoding the special case below (`tt.errMsg == "invalid instruction"`):
```suggestion
		errMsg string
		memoryProofAddresses []uint32
```